Add solidproject.org development and maintenance task force (#57)
solid/team: tasks/solidproject.org-development-maintenance.md

# solidproject.org website development and maintenance

## Responsibilities

> Members of the solidproject.org website development and maintenance are responsible for developing the website and keeping it up to date, ensuring that the architecture and the information are useful to users which accurately reflects the project's goals and values. This task force will seek feedback and report to the wider Solid Team on developments and updates via pull requests and in meetings when needed.

Reviewer: ```suggestion
Members of the solidproject.org website development and maintenance task force are responsible for developing the website and keeping it up to date, ensuring that the architecture and information are useful to users and accurately reflect the project's goals and values. This task force will seek feedback from and report to the Solid Team on developments and updates via pull requests and in meetings when needed.
```
